instrumentation/afl-llvm-pass.so.cc

```diff
@@ -715,6 +774,117 @@ bool AFLCoverage::runOnModule(Module &M) {
 
     }
 
+    if (use_threadsafe_counters) {                       /*Atomic NeverZero */
+      // handle the list of registered blocks to instrument
+      for (auto val : todo) {
```

Comment:
Is this todo variable .. fine here?

Reply:
Yes, it is just a name. It contains the parts, where new blocks are to be inserted.